src/chrome/content/rules/Pomf.se.xml

```diff
@@ -15,18 +15,28 @@
 	Fully covered subdomains:
 
 		- (www.)?	(www → ^)
+		- a
 
 -->
-<ruleset name="pomf.se (partial)">
+<ruleset name="Pomf.se">
 
 	<target host="pomf.se" />
-	<target host="*.pomf.se" />
+	<target host="www.pomf.se" />
+	<target host="a.pomf.se" />
 
 
 	<securecookie host="^\.pomf\.se$" name="^__cfduid$" />
 
 
-	<rule from="^http://(?:www\.)?pomf\.se/"
-		to="https://pomf.se/" />
+	<rule from="^http://(www\.)?pomf\.se/"
+			to="https://pomf.se/" />
 
+	<test url="http://pomf.se/" />
```

Comment:
These are redundant since they are automatically created by the target tag, so you can leave them out.